tests/test_arbitrages.py

```diff
@@ -171,39 +171,46 @@ def test_three_pool_arbitrage(get_transaction_hashes, get_addresses):
     assert arbitrage.profit_amount == first_token_out_amount - first_token_in_amount
 
 
-def test_get_all_routes():
+def test_get_shortest_route():
     # A -> B, B -> A
     start_swap = create_generic_swap("0xa", "0xb")
     end_swap = create_generic_swap("0xb", "0xa")
-    routes = _get_all_routes(start_swap, end_swap, [])
-    assert len(routes) == 1
+    route = _get_shortest_route(start_swap, [end_swap], [])
```

Comment:
```shortest_route = _get_shortest_route(...)```
sounds more aligned with what the test is doing. The following length check is meant to assert the shortest _route is 2, small comment but would be more legible. Same thing applies for the following cases and the 'route' declared in arbitrages.py

Reply:
👀

Reply:
we’re not specifically checking that it’s the shortest here though. just that a route is found

Reply:
so why is the check on the length and not on existence? A check on the length seems intended to check just that - the length of the route, not only that this route exists but that it is of a certain size

Reply:
it’s just checking it’s the route we expect. probably would be better to check like we do with the others